lib/lit/i18n_backend.rb

```diff
@@ -51,12 +51,14 @@ def lookup(locale, key, scope = [], options = {})
 
       parts = I18n.normalize_keys(locale, key, scope, options[:separator])
       key_with_locale = parts.join('.')
+      _, key_without_locale = ::Lit::Cache.split_key(key_with_locale)
 
       ## check in cache or in simple backend
       content = @cache[key_with_locale] || super
       return content if parts.size <= 1
 
-      unless @cache.has_key?(key_with_locale)
+      if !@cache.has_key?(key_with_locale) &&
+          !is_ignored_key(key_without_locale)
```

Comment:
Maybe extract it into something like

    def should_cache?(key_with_locale)
      return false if @cache.has_key?(key_with_locale)

      _, key_without_locale = ::Lit::Cache.split_key(key_with_locale)
      return false if is_ignored_key(key_without_locale)

      true
    end

? (there is no need to split keys on every `translate` call)